qa/workunits/rbd/rbd-nbd.sh

```diff
@@ -365,22 +365,31 @@ cat ${LOG_FILE}
 expect_false grep 'quiesce failed' ${LOG_FILE}
 
 # test detach/attach
-DEV=`_sudo rbd device --device-type nbd --options try-netlink map ${POOL}/${IMAGE}`
+OUT=`_sudo rbd device --device-type nbd --options try-netlink,show-cookie map ${POOL}/${IMAGE}`
+read -a DEV <<< "${OUT}"
```

Comment:
Nit: why array and not something like `read DEV COOKIE <<< "$OUT"`?  `--cookie $COOKIE` would read much nicer than `--cookie ${DEV[0]}`.

Reply:
No preference, will update.